lib/test.js

```diff
@@ -56,14 +56,14 @@ function Test (name_, opts_, cb_) {
     this._cb = args.cb;
     this._progeny = [];
     this._ok = true;
-    var depthEnvVar = 'NODE_TAPE_OBJECT_PRINT_DEPTH';
+    var depthEnvVar = process.env['NODE_TAPE_OBJECT_PRINT_DEPTH']
```

Comment:
please always use dot access instead of bracket access in javascript wherever possible.